docs/faq.md

```diff
@@ -149,7 +149,7 @@
 ## Amazon/eBay/Aliexpress jigs
 
 !!!tip ""
-    Although some Switch jigs are for sale on these websites and others like them, we have *no way of verifying that they are appropriate or safe for use with the Switch* and as such we do not recommend buying from these websites. The only seller we trust due to their reputation and our overall experience with the many jigs they have sent out is <a href="https://switchjigs.com/" target="_blank">Switch Jigs</a>.
+    Although some Switch jigs are for sale on these websites and others like them, we have *no way of verifying that they are appropriate or safe for use with the Switch* and as such we do not recommend buying from these websites.
```

Comment:
see other comment; i want a link to an stl file here as well if we're recommending 3d printing.